app/controllers/offline-controller.js

```diff
@@ -36,7 +36,7 @@ function getScriptContent() {
     // Determining hash every time, is done to make development less painful (refreshing service worker)
     // The partialScriptHash is not actually required but useful to see which offline-app-worker-partial.js is used during troubleshooting.
     // by going to http://localhost:8005/x/offline-app-worker.js and comparing the version with the version shown in the side slider of the webform.
-    const partialOfflineAppWorkerScript = fs.readFileSync( path.join( process.cwd(), 'public/js/src/module/offline-app-worker-partial.js' ), 'utf8' );
+    const partialOfflineAppWorkerScript = fs.readFileSync( path.resolve( __dirname, '../../public/js/src/module/offline-app-worker-partial.js' ), 'utf8' );
```

Comment:
I would consider using something like [pkg-dir](https://github.com/sindresorhus/pkg-dir) which is intended for this, and addresses the risk mentioned by @lognaturel.